pkgs/development/python-modules/supervisor/default.nix

```diff
@@ -22,5 +22,6 @@ buildPythonPackage rec {
     homepage = http://supervisord.org/;
     license = licenses.free; # http://www.repoze.org/LICENSE.txt
     maintainers = with maintainers; [ zimbatm ];
+    broken = true;
```

Comment:
this package builds on the branch

Reply:
oh, i forgot to be backport #70073, nevermind

Reply:
probably should backport @jonringer :+1:

Reply:
We'll backport it on this branch